app/views/posts/_editor.haml

```diff
@@ -14,13 +14,18 @@
       - elsif current_user.avatar
         = select_tag :icon_dropdown, options_for_select([[current_user.avatar.keyword, current_user.avatar.id]], current_user.avatar.id), prompt: "No Icon"
     .post-info-text
-      .post-character{class: ('hidden' unless item.character), data: {character_id: item.character_id}}= item.character.try(:name)
+      .post-character{class: ('hidden' unless item.character), data: {character_id: item.character_id || '', alias_id: item.character_alias_id || ''}}
```

Comment:
These must default to '' or the attribute can be left out of the element entirely, which might break the javascript?